### PR TITLE
Add suffixes to typos that start with "e"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -15183,6 +15222,8 @@ embdedded->embedded
 embebbed->embedded
 embedd->embed
 embeddded->embedded
+embeddder->embedder
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:15225]: Add plural?
```
embeddders->embedders
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -16875,6 +16989,7 @@ exmaines->examines
 exmaple->example
 exmaples->examples
 exmple->example
+exmples->examples
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:16992]: Adverb too?
```exmplary->exemplary```

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -16907,6 +17022,8 @@ expcetation->expectation
 expcetations->expectations
 expceted->expected
 expceting->expecting
+expcetion->exception
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:17025]: Mmmh... Shouldn't this be:
```
expcetation->expectation
```
in addition to:
```
expcetion->exception
```

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -17661,6 +17806,8 @@ extered->exerted
 extereme->extreme
 extermal->external, extremal,
 exterme->extreme
+extermely->extremely
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:17809]: Perhaps add comparative?
```
extermer->extremer
```

**Review by DimitriPapadopoulos — COMMENTED** [review]
A few suggestions...

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:17025]: `expcetation->expectation` exists a few lines earlier.

As for this entry, my script grabbed `expcet->except` from `expcet->expect, except,` and then added the suffix `-ion`.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:16996]: I added `exemplar(s)`, `exemplary`, `exemplify`+friends.
`exemplarily` `exemplariness` `exemplarity` seem too obscure to be useful.

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:17817]: Done. I also added `extermity->extremity` and its plural.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**